css/index.css

```diff
@@ -1,124 +1,221 @@
-/*   NAVBAR STYLING  */
-#navbar {
-  display: block;
-  letter-spacing: 1.5px;
-  word-spacing: 5px;
-  margin-inline: 30px;
-  margin-block: 2em;
-  text-align: right;
-}
+@import url("https://fonts.googleapis.com/css2?family=Belanosima:wght@600&family=Roboto+Mono:wght@300;400&display=swap");
 
-/*  NAV LINK STYLING  */
-a {
-  text-decoration: none;
-  font-weight: 600;
+:root {
+  --primary-color: #13435b;
+  --secondary-color: #052230;
+  --off-white: #fdfbeb;
+  --teal: #298c6f;
+  --lime: #bedb45;
+  --orange: #f97821;
 }
-a:hover, a:active {
-  color: #bedb45;
+html {
+  scroll-behavior: smooth;
+  background-color: var(--primary-color);
+  box-sizing: border-box;
 }
 
+*,
+::before,
+::after {
```

Comment:
I'm a little surprised to see these here: The only place you `::before` or `::after` content you set the padding to non-zero.

```
.skills li::before {
  content: "◦"; 
```

(Your page looks awesome, but maybe double check that this works the way you expect?)